iommu_registers.adoc

```diff
@@ -68,8 +68,10 @@ the register returns 0 and writes to that offset are ignored.
                                        CSR >>             | if `capabilities.ATS==0`
 |84    |`ipsr`          |4   |<<IPSR, Interrupt pending
                                          status register>>| No
-|88    |`iocntovf`      |4   |<<OVF, HPM counter overflows>> | if `capabilities.HPM==0`
-|92    |`iocntinh`      |4   |<<INH, HPM counter inhibits>> | if `capabilities.HPM==0`
+|88    |`iocountovf`    |4   |<<OVF, HPM counter
```

Comment:
There is still another reference to iocntovf in the iocountovf section.

Reply:
Fixed. Thanks!